src/index.html

```diff
@@ -1,13 +1,421 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Creative Bakery</title>
+    <link rel="icon" href="./images/cookie.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#" class="logo">
+              <img
+                 src="./images/logo-black.svg"
+                 alt="Bakerlab logo"
+                 class="logo__image"
+              >
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+
+
+          <div class="header__header-wrapper">
+            <p class="header__info">
+              We bake your fantasies — croissants, cakes, cookies, and cupcakes.
+            </p>
+            <div class="header__title-photo-wrapper">
+              <h1 class="header__title">
+                Creative bakery
+              </h1>
+              <div class="header__photo-container">
+                <img
+                   src="./images/header-cookie.png"
+                   alt="Header Cookie"
+                   class="header__photo"
+                  >
+              </div>
+            </div>
+
+            <a href="#about-us" class="header__button">
+                Learn more
+            </a>
```

Comment:
The content inside the tag must have only 2 spaces. It needs to be fixed everywhere
```suggestion
            <a href="#about-us" class="header__button">
              Learn more
            </a>
```